react/features/welcome/reducer.js

```diff
@@ -0,0 +1,20 @@
+import { ReducerRegistry } from '../base/redux';
+import { SET_SIDEBAR_VISIBILITY } from './actionTypes';
+
+/**
+ * Reduces the Redux actions of the feature features/recording.
+ */
+ReducerRegistry.register('features/welcome', (state = {
+    sideBarVisible: false
+}, action) => {
+    switch (action.type) {
+    case SET_SIDEBAR_VISIBILITY:
+        return {
+            ...state,
+            sideBarVisible: action.sideBarVisible
```

Comment:
rename the action parameter to just "visible"

Reply:
I explained this above